orangecontrib/prototypes/widgets/owexplpred.py

```diff
@@ -0,0 +1,409 @@
+from AnyQt.QtWidgets import (
+    QApplication, QFormLayout, QTableView,  QSplitter, QSizePolicy)
+from AnyQt.QtCore import Qt, QThread, pyqtSlot
+from PyQt5.QtGui import QSizePolicy
+import numpy as np
+import scipy.stats as st
+
+import sys
+import copy
+import logging
+import concurrent.futures
+from functools import partial
```

Comment:
The first two groups of imports should be swapped.
Python imports first, external libraries second, internal stuff last.